frappe/utils/boilerplate.py

```diff
@@ -162,6 +166,114 @@ def _create_github_workflow_files(dest, hooks):
 		f.write(github_workflow_template.format(**hooks))
 
 
+PATCH_TEMPLATE = textwrap.dedent(
+	'''
+	import frappe
+
+	def execute():
+		"""{docstring}"""
+
+		# Write your patch here.
+		pass
+'''
+)
+
+
+class PatchCreator:
+	def __init__(self):
+		self.all_apps = frappe.get_all_apps(sites_path=".")
+
+		self.app = None
+		self.app_dir = None
+		self.patch_dir = None
+		self.filename = ""
+		self.docstring = None
+		self.patch_file = None
+
+		self.fetch_user_inputs()
+		self.create_patch_file()
```

Comment:
Init seems to be doing a lot more than object initialization. Can we have these calls out?

The following seems like decent API usage too btw:

```python
pc = PatchCreator()
pc.fetch_user_inputs()
pc.create_patch_file()
```

Reply:
Makes sense.